src/blocks/illustrations/components/RewardsCircle.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { IllustrationWrapper } from '../IllustrationWrapper';
+import { IllustrationProps } from '../Illustrations.types';
+
+const RewardsCircle: FC<IllustrationProps> = (allProps) => {
+  const { svgProps: props, ...restProps } = allProps;
+  return (
+    <IllustrationWrapper
+      componentName="RewardsCircle"
+      illustration={
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width={restProps.width ?? '49'}
```

Comment:
```suggestion
          width={restProps.width ?? '48'}
```

Let's keep is consistent.